network/p2p/keyTranslator.go

```diff
@@ -35,7 +35,7 @@ func setPubKey(c elliptic.Curve, x *big.Int, y *big.Int) *goecdsa.PublicKey {
 // These utility functions convert a Flow crypto key to a LibP2P key (Flow --> LibP2P)
 
 // privKey converts a Flow private key to a LibP2P Private key
-func privKey(fpk fcrypto.PrivateKey) (lcrypto.PrivKey, error) {
+func PrivKey(fpk fcrypto.PrivateKey) (lcrypto.PrivKey, error) {
```

Comment:
making it public since this is now used for certificate generation as well.

Reply:
Not quite your PR, but should we return an error if the signing algorithm is not matched within the set of unmarshallers? e.g. `crypto.BLSBLS12381` ?